tools/libcSpecGen.cc

```diff
@@ -0,0 +1,511 @@
+#include <fstream>
```

Comment:
add a comment with author, email, and what the file is expected to do.
This way we can contact you later if we run into problems ;)